packages/material-ui/src/Dialog/Dialog.test.js

```diff
@@ -84,133 +93,118 @@ describe('<Dialog />', () => {
     assert.strictEqual(wrapper.hasClass('woofDialog'), true);
   });
 
-  it('should render Fade > div > Paper > children inside the Modal', () => {
-    const children = <p>Hello</p>;
-    const wrapper = shallow(<Dialog {...defaultProps}>{children}</Dialog>);
-
-    const fade = wrapper.childAt(0);
-    assert.strictEqual(fade.type(), Fade);
-
-    const div = fade.childAt(0);
-    assert.strictEqual(div.type(), 'div');
-
-    const paper = div.childAt(0);
-    assert.strictEqual(paper.length === 1 && paper.type(), Paper);
-
-    assert.strictEqual(paper.hasClass(classes.paper), true);
-  });
-
-  it('should not be open by default', () => {
```

Comment:
Disregarding that the very next test contradicts this it should be neither by default. `open` is required.

Reply:
😆